### PR TITLE
fix: ServiceControlPolicies feature should not fail if neither SecurityHub nor Backup feature are enabled

### PR DESCRIPTION
**Pull request by KiranGosavi**
Added condition to check if both features are set before creating resouce.

### CODE REVIEW

**Comment by sbstjn** [comment]
test pipeline worked fine.

**Comment by s0enke** [comment]
How about we push this `SCPwithSecurityAndBackup` condition down to the `ServiceControlPolicies` stack, so we could leave the special knowledge  ("SCPwithSecurityAndBackup") to the `ServiceControlPolicies` and the superwerker main stack would not need to know implementation details of the `ServiceControlPolicies` stack ([Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter)). The `ServiceControlPolicies` already knows whether SecurityHub and Backup are enabled.

**Review by s0enke — CHANGES_REQUESTED** [review]
How about we push this SCPwithSecurityAndBackup condition down to the ServiceControlPolicies stack, so we could leave the special knowledge ("SCPwithSecurityAndBackup") to the ServiceControlPolicies and the superwerker main stack would not need to know implementation details of the ServiceControlPolicies stack (Law of Demeter). The ServiceControlPolicies already knows whether SecurityHub and Backup are enabled.

**Comment by s0enke** [comment]
I would also propose to give more meaningful titles to the PRs and commit messages. E.g. instead of "issue-182" something like "fix: ServiceControlPolicies feature should not fail if neither SecurityHub nor Backup feature are enabled"

**Comment by s0enke** [comment]
> How about we push this SCPwithSecurityAndBackup condition down to the ServiceControlPolicies stack, so we could leave the special knowledge ("SCPwithSecurityAndBackup") to the ServiceControlPolicies and the superwerker main stack would not need to know implementation details of the ServiceControlPolicies stack (Law of Demeter). The ServiceControlPolicies already knows whether SecurityHub and Backup are enabled.

Actually, you did this in the initial commit 426fb63, so my comment was bogus.

File: templates/service-control-policies.yaml [hunk 1]
```diff
@@ -19,11 +19,15 @@ Parameters:
...
   IncludeBackup: !Equals [ !Ref IncludeBackup, true ]
+  SCPwithSecurityAndBackup: !And
+    - !Equals [ !Ref IncludeSecurityHub, true ]
+    - !Equals [ !Ref IncludeBackup, true ]
 
 Resources:
 
   ServiceControlPolicy:
     Type: AWS::CloudFormation::CustomResource
+    Condition: SCPwithSecurityAndBackup
```
Comment (s0enke) [inline templates/service-control-policies.yaml]: Maybe rename the condition to a more generic name like `CreateServiceControlPolicy`

**Comment by s0enke** [comment]
@KiranGosavi thanks! I cleaned up the PR so it only contains your inital commit and a small wording commit from me. Merging now

**Pull request merged**